src/screens/PasswordRecovery/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react'
+import EmailForm from './componentes/EmailForm'
+import { isValidEmail } from '../authentication/utils'
+import * as firebase from 'firebase'
+import { View, KeyboardAvoidingView } from 'react-native'
+import { SvgXml } from 'react-native-svg'
+import { Title, Description } from '~/components/texts'
+import styles from './styles'
+import Image from './resources/recover.svg'
+import GoBackToSignIn from './componentes/GoBackToSignIn'
+import NavigationService from '~/navigationService'
+
+const PasswordRecovery = () => {
+  const [error, setError] = useState(null)
+  let emailSent = false
+
+  const handleSubmit = async (values) => {
+    setError(null)
+    let email = null
+    var auth = firebase.auth()
+    if (isValidEmail(values.email)) {
+      email = values.email
+      auth.sendPasswordResetEmail(email).then(function () {
+        emailSent = true
+        NavigationService.navigate('Login', { emailSent })
+      }).catch(function (error) {
+        setError(error)
+      })
+    } else {
+      setError(true)
+    }
+  }
```

Comment:
- Don't use unnecessary variables.
- Never declare a variable with `var`. Use `let` or `const`.
- If you're in an `async` function, use `await` instead of `.then/.catch`.

The following code is much more readable:
```suggestion
  const handleSubmit = async (values) => {
    if (!isValidEmail(values.email)) {
      setError(true)
    } else {
      setError(null)
    }
    
    try {
      await firebase.auth().auth.sendPasswordResetEmail(values.email)
      NavigationService.navigate('Login', { emailSent: true })
    } catch (err) {
      setError(err)
    }
  }
```